### ISSUE TITLE
Issue compiling Rocky 8.5 Step 4 #cmake --preset container-release

### ISSUE THREAD

**Issue by bodibod** [dreamworksanimation/openmoonray#51]
Hello, I am building against Rocky 8.5 and I get this error when doing the # cmake --preset container-release

CMake Error at /installs/cmake-3.23.1-linux-x86_64/share/cmake-3.23/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Libuuid (missing: Libuuid_LIBRARY Libuuid_INCLUDE_DIRS)
Call Stack (most recent call first):
  /installs/cmake-3.23.1-linux-x86_64/share/cmake-3.23/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  cmake_modules/cmake/FindLibuuid.cmake:14 (find_package_handle_standard_args)
  arras/arras4_core/CMakeLists.txt:34 (find_package)

**Comment by bodibod** [comment]
I installed and check the cmake in the folder and it looks okay.
I am doing the Centos7 method.
I did the Docker last night and it seems it worked but I am unable to load moonray in Houdini via that method.
So I backed to the regular one which I am more use to anyway.

thanks !

Tanguy

**Comment by rgwilson68** [comment]
libuuid should be installed by the third line in step 1:

yum install -y bison flex  wget git python3 python3-devel patch giflib-devel libmng libtiff-devel libjpeg-devel libatomic libcgroup-devel libuuid-devel openssl-devel curl-devel

Is it possible that you didn't copy the entire line?

**Comment by bodibod** [comment]
Ah, good point, I think I did but I am on Rocky 8.5 and some of the dep are not 1 0 1 so I had to tweak and I maybe tweak too much ^^

Let me try again

**Comment by bodibod** [comment]
okay so it is installed, what is not going through is the libcgroup-devel

[root@render-1 tang]# yum install -y bison flex  wget git python3 python3-devel patch giflib-devel libmng libtiff-devel libjpeg-devel libatomic libcgroup-devel libuuid-devel openssl-devel curl-devel
Last metadata expiration check: 1 day, 14:38:39 ago on Sun 19 Mar 2023 09:36:58 PM PDT.
Package wget-1.19.5-10.el8.x86_64 is already installed.
Package git-2.31.1-3.el8_7.x86_64 is already installed.
Package python36-3.6.8-38.module+el8.5.0+671+195e4563.x86_64 is already installed.
Package patch-2.7.6-11.el8.x86_64 is already installed.
Package libmng-2.0.3-7.el8.x86_64 is already installed.
Package libatomic-8.5.0-16.el8_7.x86_64 is already installed.
No match for argument: libcgroup-devel
Package libuuid-devel-2.32.1-39.el8_7.x86_64 is already installed.
Error: Unable to find a match: libcgroup-devel

**Comment by cyburdine** [comment]
hey Tanguy, In order to install libcgroup-devel you may need to enable the devel repo temporarily.  I found I needed to do this for both libcgroup-devel and giflib-devel on Rocky 8.7.  

dnf -y --enablerepo=devel install libcgroup-devel giflib-devel

Hope that helps

**Comment by cyburdine** [comment]
Not sure if it will help, but I have instructions and an ansible playbook that should get you through building all of the dependencies.  check out [#20](https://github.com/dreamworksanimation/openmoonray/issues/20)
